plugins/repository-azure/build.gradle

```diff
@@ -56,6 +56,23 @@ dependencies {
   api "io.netty:netty-transport-native-unix-common:${versions.netty}"
   implementation project(':modules:transport-netty4')
   api 'com.azure:azure-storage-blob:12.23.0'
+  api 'com.azure:azure-identity:1.11.4'
+  // Start of transitive dependencies for azure-identity
+  api 'com.microsoft.azure:msal4j-persistence-extension:1.2.0'
+  // Both azure-identity:1.11.4 and msal4j-persistence-extension:1.2.0 has compile dependency on different versions of jna-platform,
+  // selected the higher version which is 5.13.0
+  api 'net.java.dev.jna:jna-platform:5.13.0'
+  api 'com.microsoft.azure:msal4j:1.14.3'
+  api 'com.nimbusds:oauth2-oidc-sdk:11.9.1'
+  api 'com.nimbusds:nimbus-jose-jwt:9.37.3'
+  api 'com.nimbusds:content-type:2.3'
+  api 'com.nimbusds:lang-tag:1.7'
+  // Both msal4j:1.14.3 and oauth2-oidc-sdk:11.9.1 has compile dependency on different versions of json-smart,
+  // selected the higher version which is 2.5.0
+  api 'net.minidev:json-smart:2.5.0'
+  api 'net.minidev:accessors-smart:2.5.0'
+  api 'org.ow2.asm:asm:9.3'
```

Comment:
Also, for ones that needed (asm, jna,...) please use the versions from `buildSrc/version.properties`, for example:
```
api "org.ow2.asm:asm:${versions.asm}"
```

Reply:
Thank you @reta, I have amended the dependency for `asm`. However, `buildSrc/version.properties` only have `jna` and no `jna-platform`. All the other ones is not in `buildSrc/version.properties`.